src/Cosi-BLS/block.lisp

```diff
@@ -185,7 +185,7 @@ added to the blockchain."
             (cosi/proofs:public-key-to-address public-key)))
          (transactions (list genesis-transaction))
          (block (create-block nil nil nil nil transactions)))
-    (cosi-simgen:node-dispatcher :genesis-block :blk block)
+    ;(cosi-simgen:node-dispatcher :genesis-block :blk block)
```

Comment:
I understand this is intended to eliminate duplicate genesis block problems, but now it does not correspond to the documentation. Don't we want the genesis block to be created on the genesis node, the originating node of the blockchain?  It seems that when you do that, the problem is that broadcast-message does not notice it's been done on that node already, and creates it anew.  Maybe that should be fixed. Or maybe the way it gets created is not done exactly right.  I think we should go back and fix it better.

Reply:
The genesis block is not necessarily created by a single node.  It is simply a valid block that all nodes can begin their computation from.  To actually have a leader orchestrating the preparation, witnessing, and signing the block via network communication is probably more complicated than it needs to be.